src/reducers/tasks.ts

```diff
@@ -0,0 +1,39 @@
+import { Reducer } from 'redux'
+import { TasksState } from 'src/reducers/types'
+
+import stringTypes from 'src/actions/stringTypes'
+import { Actions } from 'src/actions/types'
+import {
+  getTasks,
+  addTask,
+  toggleDone,
+  clearAll,
+  failure,
+} from 'src/actions/ActionHandlers'
+
+const initial_state: TasksState = {
+  tasks: [],
+  error: '',
```

Comment:
maybe its better to use `undefined` what do you think?

Reply:
do you mean that i use undefined for `error` or for the initial state?